pyUltroid/fns/openaihelper.py

```diff
@@ -0,0 +1,239 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~# SOL UserBot #~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+# sqlalchemy
+# openai
+# fake_useragent
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~#
+# Special Credits: @LucifSD02 <Telegram>
+# Special Credits: @SoulOfSukuna <Telegram>
+# Special Credits: @TrueSaiyan <Telegram>
+
+import json
+import os
+
+import openai
+import requests
+from fake_useragent import UserAgent
+from PIL import Image, ImageColor, ImageDraw, ImageFilter, ImageFont, ImageOps
+from sqlalchemy import Column, String, UnicodeText
```

Comment:
why sqlalchemy? Ultroid gives the user flexibility to choose their own database, and that is the reason why we have `udB`, which chooses a DB as required by the user.
using sqlalchemy directly here just invalidates that.

Reply:
To be honest I don't think it needs this. I can remove the file I'm pretty sure this was an old  file for something else thank you